From 161e070cd78b1398555de0efb49e151f6a4b2a33 Mon Sep 17 00:00:00 2001
From: Adrian Alonso <adrian.alonso@nxp.com>
Date: Fri, 29 Jun 2018 11:13:02 +0900
Subject: [PATCH 4072/5242] MLK-18704-2: ARM64: dts: freescale: imx8mm evk
 enable led status

commit  7b6a872c114df00b5c7ba1a51aa5aa3b473b2be5 from
https://source.codeaurora.org/external/imx/linux-imx.git

Enable gpio led status on imx8mm evk cpu board
set ON as default state

Signed-off-by: Adrian Alonso <adrian.alonso@nxp.com>
Reviewed-by: Anson Huang <Anson.Huang@nxp.com>
(cherry picked from commit e63553bbab70c06454272e37867c15a807759702)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts |   18 ++++++++++++++++++
 1 file changed, 18 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts
index 2ac3eaf..bb074d7 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts
@@ -25,6 +25,18 @@
 		stdout-patch = &uart2;
 	};
 
+	leds {
+		compatible = "gpio-leds";
+		pinctrl-names = "default";
+		pinctrl-0 = <&pinctrl_gpio_led>;
+
+		status {
+			label = "status";
+			gpios = <&gpio3 16 0>;
+			default-state = "on";
+		};
+	};
+
 	modem_reset: modem-reset {
 		compatible = "gpio-reset";
 		reset-gpios = <&gpio2 6 GPIO_ACTIVE_LOW>;
@@ -168,6 +180,12 @@
 			>;
 		};
 
+		pinctrl_gpio_led: gpioledgrp {
+			fsl,pins = <
+				MX8MM_IOMUXC_NAND_READY_B_GPIO3_IO16	0x19
+			>;
+		};
+
 		pinctrl_i2c1: i2c1grp {
 			fsl,pins = <
 				MX8MM_IOMUXC_I2C1_SCL_I2C1_SCL			0x400001c3
-- 
1.7.9.5

